rfc/0010-rancher-integration-policies.md

```diff
@@ -0,0 +1,162 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Rancher integration of Kubewarden Policies  |
+| Start Date   | 2022-08-22                       |
+| Category     | Development                      |
+| RFC PR       | https://github.com/kubewarden/rfc/pull/11  |
+| State        | **ACCEPTED**                     |
+
+
+# Summary
+[summary]: #summary
+
+This RFC explains how the Kubewarden policies, either owned by the Kubewarden
+team or 3rd party ones, can be integrated in Rancher Explorer.
+
+For policies owned by the Kubewarden team  it proposes a development workflow
+using git branches and automation. The needed Rancher Helm charts and Rancher
+Packages artifacts metadata is tracked in a separate orphan branch for each
+artifact's repo, and submitted to https://github.com/rancher/charts.
+
+For 3rd party policies not owned by the Kubewarden team,
+it proposes using the Artifact Hub API to poll for the metadata stored in
+`policy/artifacthub-pkg.yml`.
+
+
+# Motivation
+[motivation]: #motivation
+
+Problems to solve:
+
+- Obtain the list of available policies regardless of where they come from, and
+  their metadata to show in Rancher Explorer UI.
+- Allow for groups of policies to be installed.
+- Implement it efficiently and with low development maintenance costs.
+- Allow for airgap installations.
+
+## Examples / User Stories
+[examples]: #examples
+
+2. As a user, I want to install one Kubewarden policy owned by the
+   Kubewarden/Rancher team via Rancher Explorer UI.
+3. As a user, I want to install several Kubewarden policies owned by the
+   Kubewarden/Rancher team via Rancher Explorer UI.
```

Comment:
This looks like something that could be done via the UI and a process of selection boxes, like with webmails where you select multiple messages and then do a "delete/archive'.

Maybe you mean something like:

```suggestion
3. As a user, I want to install a collection of Kubewarden policies owned by the
   Kubewarden/Rancher team via the Rancher Explorer UI.
4. As a Kubewarden developer, I want to allow users to deploy a collection of related policies
   in a simple way. For example: "PSP best of", "Security best practices",...
```